using-mobile-extensions/adobe-audience-manager/audience-manager-api-reference.md

```diff
@@ -177,68 +105,6 @@ Audience.getVisitorProfile { (visitorProfile, error) in
 }];
 ```
 {% endtab %}
-
-{% tab title="iOS \(ACP 2.x\)" %}
-The `getVisitorProfile` API returns the most recently obtained visitor profile. For easy access across multiple launches of your app, the visitor profile is saved in `NSUserDefaults`. If no signal has been submitted, nil is returned.
-
-{% hint style="info" %}
-The `getVisitorProfileWithCompletionHandler` method was added in ACPAudience version 2.1.0.
-{% endhint %}
-
-### Objective-C
-
-**Syntax**
-
-```text
-+ (void) getVisitorProfile: (nonnull void (^) (NSDictionary* __nullable visitorProfile)) callback;
-
-+ (void) getVisitorProfileWithCompletionHandler: (nonnull void (^) (NSDictionary* __nullable visitorProfile, NSError* __nullable error)) completionHandler;
-```
-
-**Example**
-
-```text
-[ACPAudience getVisitorProfile:^(NSDictionary* visitorProfile){
-  // handle the visitorProfile here
-}];
-
-[ACPAudience getVisitorProfileWithCompletionHandler:^(NSDictionary * _Nullable visitorProfile, NSError * _Nullable error) {
-  if (error) {
-    // handle error here
-  } else {
-    // handle the returned visitorProfile here
-  }
-}];
-```
-
-### Swift
-
-**Example**
-
-```swift
-ACPAudience.getVisitorProfile { (visitorProfile) in
-  // handle the visitorProfile here
-}
-
-ACPAudience.getVisitorProfile { (visitorProfile, error) in
-  if let error = error {
-    // handle error here
-  } else {
-    // handle the returned visitorProfile here
-  }
-}
-```
-{% endtab %}
-
-{% tab title="React Native" %}
-### JavaScript
-
-**Example**
-
-```jsx
-ACPAudience.getVisitorProfile().then(profile => console.log("AdobeExperienceSDK: Visitor Profile: " + profile));
-```
-{% endtab %}
 {% endtabs %}
 
 ## registerExtension
```

Comment:
For the below description,  it mentions about ACP. 
Should we change to something like
The `registerExtension()` API registers the audience extension with the Mobile Core extension. This API allows the extension to send and receive events to and from the Mobile SDK.

Reply:
I'll fix this, ty.